operators/tokenizer/sentencepiece_decoder.hpp

```diff
@@ -41,6 +42,27 @@ struct KernelSentencepieceDecoder {
     std::transform(p_ids, p_ids + ids.NumberOfElement(),
                    std::back_inserter(tids),
                    [](auto _id) { return static_cast<int>(_id); });
+    if (fairseq.has_value() && (*fairseq)) {
```

Comment:
nit: '(' around *fairseq seems unnecessary.

Reply:
absolutely - likely replaced something inside the () with *fairseq and forgot to remove it, will fix it in the next PR.